corehq/apps/accounting/migrations/0053_app_user_profiles_advanced.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.privileges import APP_USER_PROFILES
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
```

Comment:
Nit: This isn't grandfathering privileges, right? It's adding new privileges to existing accounts, not ensuring existing accounts retain privileges that would otherwise be taken away

Reply:
i think we have always referred to this process also as "grandfathering" in the past